package.json

```
@@ -82,7 +81,7 @@
     "@vue/babel-preset-app": "4.5.15",
     "@vue/cli-plugin-babel": "4.5.15",
     "@vue/cli-service": "4.5.15",
-    "@vue/compiler-sfc": "3.1.5",
+    "@vue/compiler-sfc": "^3.2.31",
```

Can you specify the version directly? Same as the below.

```suggestion
    "@vue/compiler-sfc": "3.2.31",
```